applications/stream-applications-core/stream-applications-security-common/src/main/java/org/springframework/cloud/stream/app/security/common/AppStarterWebSecurityAutoConfiguration.java

```diff
@@ -126,4 +127,38 @@ protected void configure(HttpSecurity http) throws Exception {
 			}
 		};
 	}
+	/**
+	 * Extends {@link EndpointRequest} to allow HTTP methods to be specified on the request matcher.
+	 */
+	static class MethodAwareEndpointRequest {
+
+		/**
+		 * Returns a matcher that includes the specified {@link Endpoint actuator endpoints} and http method.
+		 * For example: <pre class="code">
+		 * EndpointRequest.to(HttpMethod.POST, "loggers")
+		 * </pre>
+		 * @param httpMethod the http method to include
+		 * @param endpoints the endpoints to include
+		 * @return the configured {@link RequestMatcher}
+		 */
+		static RequestMatcher to(HttpMethod httpMethod, String... endpoints) {
+			final EndpointRequest.EndpointRequestMatcher matcher = EndpointRequest.to(endpoints);
+			return (request) -> {
```

Comment:
**[TinyNit]** we could make this a bit more DRY and create a private helper that takes in a request matcher guarded by httpmethod which would then be called by both of these. Not a big deal either way though.